### DISCUSSION TITLE
1.1.4 Texture2D.GetPixels 호출시 컬러값이 undefine

### DISCUSSION THREAD

**Discussion by x4cf** [naverz/zepeto-studio-kor#385] [Issue Report (Legacy)]
Texture2D.GetPixels 호출시 컬러값이 undefine 으로 가져와집니다

택스처의 픽셀 개수는 잘 가져와요

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 지금도 같은 오류가 발생하나요? 관련 코드나 구조를 첨부해 주시면 더 정확한 도움을 드릴 수 있을것 같습니다.
